contrib/nodejs/buffer.js

```diff
@@ -55,3 +55,11 @@ buffer.SlowBuffer.prototype.slice;
  * @return {string}
  */
 buffer.SlowBuffer.prototype.toString;
+
+/**
+ * @param {number} size
+ * @param {(string|Buffer|number)=} fill
+ * @param {number=} encoding
```

Comment:
According to https://nodejs.org/api/buffer.html#buffer_class_method_buffer_alloc_size_fill_encoding, I think this should be `string=`?

Reply:
Oh, yes, good catch.